docs/app.md

```diff
@@ -407,6 +408,12 @@ FLAG DESCRIPTIONS
     will be used for this.
 
     You can also set this value by setting the MITTWALD_SSH_USER environment variable.
+
+  --sub-directory=<value>  specify a sub-directory within the app installation to download
+
+    This is particularly useful when you only want to download a specific sub-directory of the app installation, for
+    example when you are using a deployment tool that manages the app installation directory itself, and you only want
+    to download exempt files, like environment specific configuration files or user data.
```

Comment:
It might be helpful to give an example. If I know the file hierarchy on the server, it might be helpful to have this in the description:

```suggestion
  --sub-directory=<value>  specify a sub-directory within the app installation to download

    This is particularly useful when you only want to download a specific sub-directory of the app installation, for
    example when you are using a deployment tool that manages the app installation directory itself, and you only want
    to download exempt files, like environment specific configuration files or user data. For example if you want to download `html/my-app-12345/config` set `--sub-directory=config`
```

Reply:
Fixed.